docs/hub/oauth.md

```diff
@@ -0,0 +1,45 @@
+# Sign in with Hugging Face
+
+Besides [spaces](https://huggingface.co/docs/hub/spaces-oauth), it's also possible to create an OAuth / OpenID connect app for your website. This will allow users to sign-in to your website using their HF account.
+
+It's also possible to have the additional `email` scope.
+
+Contact us at [website@huggingface.co](mailto:website@huggingface.co) with the desired name, logo url, scope and redirect urls for the OAuth app. We will provide you with the client ID and client secret.
+
+All other information is available in the [OpenID metadata](https://huggingface.co/.well-known/openid-configuration).
+
+## Branding
+
+You are free to use the Hugging Face logo in the "Sign in with Hugging Face" button.
+
+Here's an example button in HTML+CSS:
+
```

Comment:
Maybe add an example of how the snippet of code renders on a website? Even if it's just an image I think it's nice to have an idea of what to expect.

Reply:
Instead of complex CSS, you can tell to use a badge maybe (in a <a>)?
https://huggingface.co/datasets/huggingface/badges#sign-in-with-hugging-face